suites/tipset/rewards_penalties.go

```diff
@@ -208,6 +209,56 @@ func TestMinerRewardsAndPenalties(t *testing.T, factory state.Factories) {
 		td.AssertBalance(builtin.BurntFundsActorAddr, big.Add(halfBalance, gasPenalty))
 	})
 
+	t.Run("insufficient gas to cover return value", func(t *testing.T) {
```

Comment:
While writing this test the following bug was found in lotus: https://github.com/filecoin-project/lotus/pull/1638. Unfortunately even with the aforementioned fix lotus doesn't pass this test, and go-filecoin doesn't compile against this version of chain-validation, so I cannot generate the resource files for this.

I need to dig in more and see why lotus fails, from the initial debugging I have done it appears the miner is being paid 522 more FIL than it should, this is the value of [gasUnitsUsed](https://github.com/filecoin-project/chain-validation/pull/176/files#diff-01cdcbcb836fde88bea237f4d0cc0364R246)